internal/servers/worker_auth.go

```diff
@@ -2,12 +2,19 @@ package servers
 
 import (
 	"context"
+	"time"
 
 	"github.com/hashicorp/boundary/internal/errors"
 	"github.com/hashicorp/boundary/internal/servers/store"
 	"google.golang.org/protobuf/proto"
 )
 
+const (
+	// DefaultCertificateLifetime represents how long root certificates are valid for.
+	// The default is set to a year
```

Comment:
Can you add a FIXME to make it clear that the year lifetime is temporary?

Reply:
I am not sure what the non-temporary plan is. But if root certificates will each get their own lifetime when created, please also add a `FIXME` to the job as well. The job would need to behave like the Vault Token renewal job which runs more frequently and uses the expiry time in the database to determine if the certificates needs to be renewed or not rather than a blanket renewal of all tokens.